storage/sealer/sealtasks/task.go

```diff
@@ -67,7 +69,8 @@ var shortNames = map[TaskType]string{
 	TTCommit1:    "C1",
 	TTCommit2:    "C2",
 
-	TTFinalize: "FIN",
+	TTFinalize:        "FIN",
+	TTReleaseUnsealed: "FUS",
```

Comment:
RUS?

Reply:
I went with FUS (to mean FinalizeUnsealed), to be consistent with other finalization-related short names; Tho I guess the task should be called FinalizeUnsealed in that case..

Reply:
Renamed but kept some other ReleaseUnsealed in less visible places to avoid breaking the API too much; This code will realistically all get replaced when the new scheduler comes, so just a non-confusing UX is probably good enough.

Reply:
I think ideally we should keep the names of the API and tasks consistent considering thats the pattern followed for others?

Reply:
I didn't want to change the already existing method on the miner api - ReturnReleaseUnsealed was already there; This has the subtle side effect of making lotus-miner technically forward-compatible with workers (e.g. if you downgrade lotus-miner when you have newer versions of workers still running, and trying to return ReleaseUnsealed those results will get recorded correctly)

tbh it doesn't matter that much, just felt like the right approach.